tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -370,6 +375,12 @@ def run_tests(config, test_paths, product, **kwargs):
         logger.info("Tolerating %s unexpected results" % unexpected_total)
         return True
 
+    all_unexpected_passed = (unexpected_total == unexpected_pass_total)
+    if all_unexpected_passed and not kwargs["fail_on_unexpected_pass"]:
```

Comment:
If there are 0 unexpected results, I think this will unnecessarily log? (The return value will still be correct, but still :))

Reply:
Yep, good catch. Fixed.